packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/telecom-telephony-alias-portabilit-portabilities-attach.controller.js

```diff
@@ -0,0 +1,51 @@
+import endsWith from 'lodash/endsWith';
+
+angular.module('managerApp').controller('TelecomTelephonyServicePortabilityMandateAttachCtrl',
+  class TelecomTelephonyServicePortabilityMandateAttachCtrl {
```

Comment:
As always, better `export` your controller, and inject it into angular.module after